Makefile

```diff
@@ -6,24 +6,27 @@ all: test
 
 .PHONY: build/host/CMakeCache.txt
 build/CMakeCache.txt:
-	$(MAKE) rebuild-cmake
+	@$(MAKE) rebuild-cmake
 
 .PHONY: install-pkgs
 install-pkgs: rebuild-cmake
-	(cd build && ninja install-pkgs)
-	(cd tests/toit-png-tools && $(MAKE) rebuild-cmake)
-	(cd tests/toit-png-tools && cmake -DTOITRUN:FILEPATH="$${TOITRUN:-toit.run}" -DTOITPKG:FILEPATH="$${TOITPKG:-toit.pkg}" build)
-	(cd tests/toit-png-tools && $(MAKE) install-pkgs)
+	@(cd build && ninja install-pkgs)
+	@[ -f tests/toit-png-tools/Makefile ] || \
+		(echo "Toit-png-tools directory doesn't contain a makefile. Run 'git submodule update --init' to get the submodule." \
```

Comment:
It feels odd to have the directory name capitalized in the echo output. Maybe 

`"The toit-png-tools/ directory ..."`

Reply:
I also print two lines now.
done.